tests/lib/util/glob-util.js

```diff
@@ -301,17 +319,19 @@ describe("globUtil", () => {
         it("should not return a file from a glob if it matches a pattern in an ignore file", () => {
             const options = { ignore: true, ignorePath: getFixturePath("glob-util", "ignored", ".eslintignore") };
             const patterns = [getFixturePath("glob-util", "ignored", "**/*.js")];
-            const result = globUtil.listFilesToProcess(patterns, options);
 
-            assert.strictEqual(result.length, 0);
+            assert.throws(() => {
+                globUtil.listFilesToProcess(patterns, options);
+            }, `All files matched by '${patterns[0]}' are ignored.`);
         });
 
         it("should not return a file from a glob if matching a specified ignore pattern", () => {
```

Comment:
Is this test name still accurate given that we have an `assert.throws` in the test body?